### PR TITLE
Modify English documents of While and while_loop

### PR DESCRIPTION
**Pull request by gfwm2013**
根据icafe卡片[#4342](http://newicafe.baidu.com/issue/DLTP-4342/show?from=page)修改While和while_loop的英文文档

------

172 18 25 75_8088_documentation_docs_en_api_layers_While html

172 18 25 75_8088_documentation_docs_en_api_layers_while_loop html

### CODE REVIEW

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -832,6 +832,8 @@ class While(object):
         A new OP :ref:`api_fluid_layers_while_loop` is highly recommended instead of ``While`` if the shape of parameter ``cond`` is [1].
         OP :ref:`api_fluid_layers_while_loop` is easier to use and is called with less code but does the same thing as ``While`` .
 
+    Notice:
+        Local variables created in ``While`` are similar it created in while in C++, and cannot be referenced externally, As a result, they cannot be obtained through ``fetch_list`` of ``Executor``. If you want to implement this function, PaddlePaddle provides ``assign`` interface to assign local variables to external, please refer to example code 2 or refer to `issue#22724 <https://github.com/PaddlePaddle/Paddle/issues/22724>`_.
```
Comment (zhhsplendid) [inline python/paddle/fluid/layers/control_flow.py]: “are similar it” -> "are similar to that"

"while in C++, and cannot be referenced externally, As a result" ->  "``while`` of C++ and cannot be referenced externally. As a result"

"If you want to implement this function" -> "If you would like to access the variable out of ``While``"

''``assign`` interface" -> "``assign`` API".

"variables to external. Please refer"

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -928,6 +957,10 @@ def while_loop(cond, body, loop_vars, is_test=False, name=None):
     """
     while_loop is one of the control flows. Repeats while_loop `body` until `cond` returns False.
 
+    Notice:
+        Local variables defined in ``body`` cannot be obtained through ``fetch_list`` of ``Executor`` , variables should
+        be defined outside ``body`` and placed in ``loop_vars`` for looping, then thses variables can be obtained by ``fetch_list`` .
```
Comment (zhhsplendid) [inline python/paddle/fluid/layers/control_flow.py]: "thses variables can be obtained" -> "these variables can be fetched".

File: python/paddle/fluid/layers/control_flow.py (hunk 1 above)
Comment (gfwm2013) [reply python/paddle/fluid/layers/control_flow.py]: Done.

File: python/paddle/fluid/layers/control_flow.py (hunk 2 above)
Comment (gfwm2013) [reply python/paddle/fluid/layers/control_flow.py]: Done.

**Pull request closed without merging**